### PR TITLE
ssa: verify files and firewall rules on rhevm

### PR DESCRIPTION
**Pull request by vrutkovs**
{{pytest: cfme/tests/infrastructure/test_host_analysis.py --use-provider 'rhevm34' -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ca5fbdec91411404f27d8a86a8cccaf0b6424320:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 89b8dbfed9c556cf3370909c617c590614807e2d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/infrastructure/test_host_analysis.py [hunk 1]
```diff
@@ -104,6 +104,9 @@ def is_host_analysis_finished():
     sel.handle_alert()
 
     # Check results of the analysis
+    drift_history = test_host.get_detail('Relationships', 'Drift History')
```
Comment (psav) [inline cfme/tests/infrastructure/test_host_analysis.py]: Presuming the get_detail is preferred here so that we navigate the first time?

Comment (vrutkovs) [inline cfme/tests/infrastructure/test_host_analysis.py]: Yes, ```get_detail``` will navigate to the details page, however it cannot detect if simply a refresh is needed, so in order to speed up I used ```InfoBlock``` in next occurrences

**Comment by dajoRH** [comment]
Lint report for commit faeb19d15bdb04bddb1e0372ae61903708f40e52:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/infrastructure/test_host_analysis.py [hunk 2]
```diff
@@ -58,6 +59,10 @@ def test_run_host_analysis(request, setup_provider, provider, host_type, host_na
     Metadata:
         test_flag: host_analysis
     """
+    # Rhevm cannot do analysis on upstream - not supported
```
Comment (psav) [inline cfme/tests/infrastructure/test_host_analysis.py]: Can we make this an uncollectif please

File: cfme/tests/infrastructure/test_host_analysis.py (hunk 1 above)
Comment (psav) [inline cfme/tests/infrastructure/test_host_analysis.py]: Love it!

**Comment by dajoRH** [comment]
Lint report for commit e264039b047492e4f848d8f1fcdec481f9442a33:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**